Travis multi os build (Linux/OSX)

arl opened the pull request:
this PR replaces PR #31

contains : 
- Travis-ci OSX build and deploy to GitHub Releases of cutecom-ng DMG archive
- Travis-ci Linux build and deploy to GitHub Releases of cutecom-ng deb package
- README.md additions (install instructions and travis badge)
- platform specific build scripts in `./ci/travis` folder

rasky committed:
Initial travis support

rasky committed:
We need sudo power

rasky committed:
Add debugging

rasky committed:
Switch to different PPA

rasky committed:
Copy dependencies into bin

rasky committed:
Add linux deploy scripts

rasky committed:
Mark as executable

rasky committed:
Build deb package

arl committed:
add deploy to gh-releases

arl committed:
add qt5 as deb package dependencies

arl committed:
clean/simplify travis.yml

arl committed:
include gh tag in deb package name

arl committed:
add linux install instructions

arl committed:
add travis badge

arl committed:
update README with MacOSX binary installation

aurelien-rainone commented:
@rasky could you provide or replace develersrl account encrypted github oauth token please?

arl committed:
replace my GitHub oauth token with develbot one

doing this, so that when PR #34 will be merged, travis-ci build
artefacts can be pushed onto develersrl repository.
